doc/nrf/protocols/lte/psm.rst

```diff
@@ -0,0 +1,267 @@
+.. _cellular_psm:
+
+Power saving techniques
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Cellular technology is an example of an LPWAN (Low Power Wide Area Network) because to its coverage, high mobility, roaming capabilities, and infrastructure.
+However, battery lifespan can be a problem for devices using cellular IoT if you do not fully understand the power-saving capabilities and how to use them to decrease power consumption.
+
+This guide explains following concepts for saving power:
+
+* :ref:`Radio activity modes <cellular_radio_modes>`, to enable you to examine power-saving modes in cellular IoT.
+* :ref:`cellular_power_saving`:
+
+  * Discontinuous Reception
+  * Connected mode DRX (cDRX)
+  * Idle-mode DRX (iDRX)
+  * extended Discontinuous Reception (eDRX)
+  * Power Saving Mode (PSM)
```

Comment:
Should add in a section for "Smart PSM" 
(the "marketing" name for our proprietary PSM solution that will give PSM like sleep even if the network does not grant it because of some roaming agreement for example. --> https://docs.nordicsemi.com/bundle/ref_at_commands_nrf91x1/page/REF/at_commands/nw_service/feaconf_set.html)

Reply:
The comment @tokangas made regarding QCI also applies here. We refer to it as proprietary PSM in NCS libs.

Reply:
Added a section for Proprietary Power Saving Mode (PSM) or Smart PSM here : https://github.com/nrfconnect/sdk-nrf/pull/21280/commits/3eb4e1836b981d2263502e69266f195a52fe3444#diff-a971431a92fa32e3999ac61552d912566f51537d15fd98a1b3d6783cab23b19bR126 . Please review.